packages/edit-navigation/src/filters/remove-edit-unsupported-features.js

```diff
@@ -15,7 +15,7 @@ const removeNavigationBlockEditUnsupportedFeatures = createHigherOrderComponent(
 				{ ...props }
 				hasSubmenuIndicatorSetting={ false }
 				hasItemJustificationControls={ false }
-				hasListViewModal={ false }
+				hasListViewModal={ true }
```

Comment:
I think `true` is the default, so could just omit this prop.

The `hasListViewModal` code could also be removed from the navigation block's edit function, as I think this is the only place it is used.

Reply:
I removed `hasListViewModal` in 1efb5f43a28f367e3e458c7599c2a310a2ce590a Thanks for suggesting this.